### PR TITLE
Windows support added to setup.py

### LINKED ISSUE #32: Pip installation fails on windows
I downloaded beprof code from git and trying to install it locally using pip (Python3, anaconda, Windows 10), but installation fails:

```
C:\Users\User\PycharmProjects\beprof>pip install . --user
Processing c:\users\user\pycharmprojects\beprof
Collecting numpy>=1.10.4 (from beprof==0.1.0)
  Using cached numpy-1.10.4.tar.gz
Collecting scipy>=0.16.0 (from beprof==0.1.0)
  Using cached scipy-0.17.0.tar.gz
(......)
    error: Microsoft Visual C++ 10.0 is required (Unable to find vcvarsall.bat).

    ----------------------------------------
Command "C:\Users\User\Anaconda3\python.exe -c "import setuptools, tokenize;__file__='C:\\Users\\User\\AppData\\Local\\Temp\\pip-build-g_bmb6r6\\numpy\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record C:\Users\User\AppData\Local\Temp\pip-yddb2uuq-record\install-record.txt --single-version-externally-managed --compile --user --prefix=" failed with error code 1 in C:\Users\User\AppData\Local\Temp\pip-build-g_bmb6r6\numpy
You are using pip version 7.1.2, however version 8.0.2 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

```

### LINKED ISSUE #42: Pip install from git fails
```
ant6% ➜ pip install git+https://github.com/grzanka/beprof.git@master 
Collecting git+https://github.com/grzanka/beprof.git@master
  Cloning https://github.com/grzanka/beprof.git (to master) to /tmp/pip-aghy__07-build
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "/tmp/pip-aghy__07-build/setup.py", line 4, in <module>
        import versioneer
    ImportError: No module named 'versioneer'

    During handling of the above exception, another exception occurred:

    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-aghy__07-build/setup.py", line 9, in <module>
        subprocess.call(["versioneer", "install"])
      File "/usr/lib64/python3.4/subprocess.py", line 537, in call
        with Popen(*popenargs, **kwargs) as p:
      File "/usr/lib64/python3.4/subprocess.py", line 858, in __init__
        restore_signals, start_new_session)
      File "/usr/lib64/python3.4/subprocess.py", line 1456, in _execute_child
        raise child_exception_type(errno_num, err_msg)
    FileNotFoundError: [Errno 2] No such file or directory: 'versioneer'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-aghy__07-build/
```

### PR DESCRIPTION
**Pull request by grzanka**
Closes #32

### CODE REVIEW

**Comment by grzanka** [comment]
@ant6 can you review and merge when ready ?

**Comment by antnieszka** [comment]
Ok

**Comment by grzanka** [comment]
BTW - travis is slow again: https://www.traviscistatus.com/incidents/nf3tg08pqg2d

File: setup.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def setup_versioneer():
             # find path relative to main_path where
             # *versioneer binary* was installed
             bin_path = [x[len("  "):] for x in output.splitlines()
-                        if x.endswith( os.path.sep + "versioneer")][0]
+                        if x.endswith(os.path.sep + "versioneer")][0]
```
Comment (antnieszka) [inline setup.py:60]: :+1:

**Comment by antnieszka** [comment]
This build won't end -.-'

**Comment by grzanka** [comment]
Be patient - by tomorrow should be ready. In the meantime you can try to make some tests with wercker (#67)

**Comment by antnieszka** [comment]
Ugh, test passed but I miss-clicked... now they won't, because we don't have wercker.yml here :/ Can I just merge it...? Test passed, really :P

**Comment by grzanka** [comment]
I'd like to avoid merging stuff that don't pass tests. 
What if first wercker is added in other branch (with proper PR), then this PR can be rebased (--force) and tomorrow should pass tests, as travis will be faster then ?

**Comment by antnieszka** [comment]
Ok, maybe we should merge wercker with minimal config, like 'run pytest and do nothing more'? They we could rebase this branch on new master.

**Comment by grzanka** [comment]
@ant6 I've just rebased and squashed. Please merge if all tests are OK.

**Comment by antnieszka** [comment]
I don't get it - it was squashed and now it seems it's not - looking at commits: https://github.com/DataMedSci/beprof/commits/master Was it supposed to look like this or I have to squash again when merging?

**Comment by grzanka** [comment]
After merging PR #79 only following commits have appeared:  
- e2a59fcb961988b5ec4e26ae1ec04dc3415a2574 - the change
- 08547da6cc0093d7e2754fac69f2d8ecaaa1f5ef - merge commit

Which commits (paste here SHAs) seems strange to you ?

**Comment by antnieszka** [comment]
I just thought it should look more like this commit:
https://github.com/DataMedSci/beprof/commit/a6f1eede1bee508d3d15811fded1a0a0c4e87331

But I have another question - when merging - should I use Squas merge Github suggests? Or the typical?

**Comment by grzanka** [comment]
I prefer to use:
- "squash merge" when there are many commits AND it is not necessary to keep all of them\
- "normal merge" when there is single commit OR many commits and makes sense to keep their history (changes come in few steps and they are well described)

But still it is better to read more about it.

**Pull request merged**